docker-machine-nfs.sh

```diff
@@ -146,15 +146,15 @@ configureNFS()
     echoSuccess "\t\t\t\t\t\tOK"
 }
 
-# @info:    Configures the Boot2Docker to mount nfs
-configureBoot2Docker()
+# @info:    Configures the Docker Machine to mount nfs
+configureDockerMachine()
```

Comment:
Actually this does really configure Boot2Docker and not docker-machine, so here the naming was correct imho.